Gemfile

```diff
@@ -7,7 +7,6 @@ ruby '2.3.0'
 gemspec
 
 group :development, :test do
-  gem 'rspec-collection_matchers'
```

Comment:
Yup. Just another convention : stuff that is critical for testing should go in the gemspec, and the rest (linters, guard, etc.) should not. In our case, as rubocop is included in Rakefile already, it makes no longer sense not to add it to gemspec.